src/get-aws-results.ts

```diff
@@ -72,10 +72,22 @@ export const getAwsResults = async (
        * no idea what service is being requested. You're just lucky this isn't the v3 AWS SDK
        * where there's a chance we're going to end up doing an eval so we still be service agnostic
        */
-      const client = new (<any>AWS)[service](clientParams);
+      let client;
```

Comment:
Is it possible (I cant remember) for this to still be const as youre only assigning a value once, not re-assigning it?

Reply:
It needs to be defined outside the try - this was how I thought it should be done.

Reply:
I agree the construct is correct, I'm just querying let vs. const. In JS you can use const in places it doesn't immediately seem like you could, and you should always use const where you can. I can't remember if the linter picks this up or not, but in short, if const works use const, otherwise let is right.